lib/Receiver.js

```diff
@@ -131,10 +131,6 @@ class Receiver {
    */
 
   expectHeader(length, handler) {
-    if (length == 0) {
-      handler(null);
-      return;
-    }
     this.expectBuffer = this.headerBuffer.slice(this.expectOffset, this.expectOffset + length);
```

Comment:
Unrelated but I'm not sure why slice is called with `this.expectOffset` and `this.expectOffset + length`. I think it is safe to use `0` and `length` respectively. Also I _think_ the `headerBuffer` on line 56 could be of length 8 instead of 10.